datacommons/utils.py

```diff
@@ -49,6 +51,25 @@
 # The default value to limit to
 _MAX_LIMIT = 100
 
+# Environment variables used by the package
+_ENV_VAR_API_KEY = 'DC_API_KEY'         # The API KEY
```

Comment:
should this be None by default?

Reply:
Ah this is the environment variable name, not the value itself. I can clarify in the code